core/src/main/java/ch/cyberduck/core/AbstractProtocol.java

```diff
@@ -178,11 +304,19 @@ public String getHostnamePlaceholder() {
 
     @Override
     public String getUsernamePlaceholder() {
+        return getDefaultUsernamePlaceholder();
+    }
+
+    private static String getDefaultUsernamePlaceholder() {
```

Comment:
Why is this required?

Reply:
No need to match duplicated `LocaleFactory.localizedString("Username", "Credentials");` in multiple places.

Reply:
There is a single occurrence only and such duplicated code does no harm.